components/cam/src/physics/cam/micro_p3_interface.F90

```diff
@@ -774,7 +779,7 @@ subroutine get_cloud_fraction(its,ite,kts,kte,ast,qc,qr,qitot,method, &
           ! then leave rcldm as cloud fraction at current level
```

Comment:
Suggest adding a comment here: "max overlap is the max cloud fraction in all layers above which are connected to this one by a continuous band of precip mass. If there's no precip mass falling into a cell, its precip frac is equal to the cloud frac, which is probably ~zero."

Reply:
yep, I've added this text to the file.